src/sentry/issues/auto_source_code_config/in_app_stack_trace_rules.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from collections.abc import Mapping, Sequence
+from typing import Any
+
+from sentry.issues.auto_source_code_config.code_mapping import CodeMapping
+from sentry.issues.auto_source_code_config.utils import PlatformConfig
+from sentry.models.project import Project
+from sentry.utils import metrics
+
+from .constants import METRIC_PREFIX
+
+logger = logging.getLogger(__name__)
+
+OPTION_KEY = "sentry:automatic_grouping_enhancements"
```

Comment:
What would you think of `derived_grouping_enhancements` as an option name rather than `automatic_grouping_enhancements`? 

If I didn't know anything about how this worked, my first thought seeing the description "auto enhancements" would be that they were enhancements applied automatically. OTOH, that's not a lie - we do apply them automatically. OTOH, we also apply the default and custom enhancements automatically, so under my gut-reaction interpretation "automatic" isn't a great differentiator.

Reply:
Works for me 👍🏻